module-4/backend/week_one.md

```diff
@@ -5,15 +5,46 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+How objects are constructed and used in Javascript.
+
 2. What are some tools to help debug JavaScript code?
+
+The Debugger, Chrome developer tools.
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+Mocha, Chai
+
 4. What is the syntax for invoking a function?
+
+functionName();
+
 5. What's `this` in JavaScript?
+
+In a general sense 'this' is a way of refering it the parent element of a JS element, but differs depending on the context. Inside of an object, this refers to the object. Outside of an object, it will depend on the context but this can refer to the window.
+
 6. What is Webpack and why is it useful?
+
+Webpack is a tool that compiles JS and CSS assets. It is useful because it allows us to have separate files for JS and CSS that it will then import into one file. 
+
 7. When/why do you want to use event delegation?
+
+You might want to use event delegation when you want a parent or grandparent element to be clickable but still want to know which child element was actually clicked. 
```

Comment:
absolutely - or if you want to bind an event to an element that is loaded dynamically (aka not on the page on document ready)